Devscord.DiscordFramework/Middlewares/Contexts/ReactionContext.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Devscord.DiscordFramework.Framework.Architecture.Middlewares;
+
+namespace Devscord.DiscordFramework.Middlewares.Contexts
+{
+    public class ReactionContext : IDiscordContext
+    {
+        public ulong ChannelId { get; }
+        public string ChannelName { get; }
+        public string EmoteName { get; }
+        public string MessageContent { get; }
+        public string MessageAuthorName { get; }
```

Comment:
tu powinien być UserContext

Reply:
EmoteName też? To bym zostawił normalnie.